### DISCUSSION TITLE
How to make many to many tables visible [Apr 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1805] [Discord Threads]
> 🙋‍♂️ **Harbard**  
## Some tables are not shown in NocoDB, even if they are detected. I can't see micro/macrozones tables in noco, am I missing something here ? (I don't know if should make a Github issue or not)  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/963411547902844938/unknown.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### right top menu > themes > show m2m tables  
#  
> 🙋‍♂️ **Harbard**  
#### Oh, as simple as that, tyvm ! <a:prayge:885959642482610177>  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/963411548859162754)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
